zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/Collection.java

```diff
@@ -1325,5 +1322,29 @@ public long getPublishTimeMilliseconds() {
         }
         return 0;
     }
+
+    private static BiFunction<String, String, String> COLLECTION_CONTENT_PATH = (collectioName, uri) -> {
```

Comment:
I don't think this needs to be a function it can be a static method.
```
private static String collectionContentPath(String collectioName, String uri) {....}
```